dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/data/AbstractJdbcEventAnalyticsManager.java

```diff
@@ -268,6 +269,16 @@ else if ( ValueType.COORDINATE == queryItem.getValueType() )
 
                 columns.add( coordSql );
             }
+            else if ( ValueType.ORGANISATION_UNIT == queryItem.getValueType()
+                && params.getCoordinateField().equals( queryItem.getItem().getUid() ) )
+            {
+                String colName = quote( queryItem.getItemId() + OU_GEOMETRY_COL_SUFFIX );
+
+                String coordSql =  "'[' || round(ST_X(ST_Centroid(" + colName + "))::numeric, 6) || ',' || round(ST_Y(ST_Centroid(" + colName + "))::numeric, 6) || ']' as " + colName;
```

Comment:
This comes back to my previous question. Do we only support Point for OrgUnit Geometry? I would expect org units most often to be areas? Also, will this result in the correct numbers? I don't remember which standard Bjørn told me to use, but he mentioned a specific one.

Reply:
See comment above.